DeAceroDLabs/marketplace-frontend
# [DMP-685] Reduced card size

## What is done

[[DMP-685] Corregir tamaño cards](https://deacero.atlassian.net/browse/DMP-685)

## How is done

1. Changed min-width for medium sized cards

## How to test

1. Run the project via `npm start`
3. On home, check cards don't overflow over to the right
## Evidence

BEFORE
![image](https://user-images.githubusercontent.com/116199447/199981596-5a71eb5f-1277-4896-86d4-95b184894989.png)


AFTER
![image](https://user-images.githubusercontent.com/116199447/199981238-118a52ca-5ccb-4ecb-9fcd-cea8376fe06b.png)
